### PR TITLE
(SIMP-8904) Finalize EL6 removal

### PR DESCRIPTION
**Pull request by trevor-vaughan**
* Also fixed the version number

SIMP-8904 #close

### CODE REVIEW

File: manifests/server/params.pp [hunk 1]
```diff
@@ -85,8 +85,8 @@
   }
 
   if (
-    ($facts['os']['name'] in ['RedHat','CentOS','OracleLinux'] and versioncmp($facts['os']['release']['major'],'7') >= 0) or
-    ($facts['os']['name'] in ['Fedora'] and versioncmp($facts['os']['release']['major'],'22') >= 0)
+    ($facts.dig('os', 'name') in ['RedHat','CentOS','OracleLinux'] and versioncmp($facts.dig('os','release','major'),'7') >= 0) or
+    ($facts.dig('os','name') in ['Fedora'] and versioncmp($facts.dig('os','release','major'),'22') >= 0)
```
Comment (op-ct) [inline manifests/server/params.pp:89]: Should we keep this?  Fedora isn't in the **metadata.json**'s `operatingsystem_support` list

**Review by op-ct — APPROVED** [review]
👁️ 👍

File: manifests/server/params.pp (hunk 1 above)
Comment (lnemsick-simp) [reply manifests/server/params.pp:89]: This should be removed because `ssh::server`  has `simplib::assert_metadata( $module_name )`

**Comment by lnemsick-simp** [comment]
Are the fallback ciphers and macs appropriate in `ssh::client::params`

**Comment by lnemsick-simp** [comment]
Also, are the `ssh::client::params` has OS checks appropriate?

**Comment by lnemsick-simp** [comment]
`checks.yaml` have `operatingsystemmajrelease: '6'` confines.

**Comment by trevor-vaughan** [comment]
@lnemsick-simp OK, I *think* I've gotten everything this time :-/

**Review by lnemsick-simp — APPROVED** [review]

**Pull request merged**